packages/lime-system/files/usr/lib/lua/lime/get_ifname.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+if [ -n "$1" ] then
+    . /usr/share/libubox/jshn.sh
+
+    json_load "$(cat /etc/board.json)"
+    json_select network
+    if json_get_type Type "$1" && [ "$Type" == object ]; then
+        json_select "$1"
+        json_get_var ifname ifname
```

Comment:
Have you checked this on boards with DSA? On TL-WR941ND this can be eg. `lan1 lan2 lan3 lan4` (where each element is an independent network interface, all of them ending up in the `lan` bridge, the bridge ends up having the mac address of the first interface added to it afaik)

Reply:
I'm still in Tokyo with minimal hardware to test. How would you implement that?

Reply:
@dangowrt I tried to find some further information online but seems like the TL-WR941ND is no longer supported?

Reply:
for the usage we do returning just the first would probably be enough